packages/dx-react-grid-material-ui/src/templates/search-panel-input.jsx

```diff
@@ -0,0 +1,30 @@
+import * as React from 'react';
+import * as PropTypes from 'prop-types';
+import classNames from 'classnames';
+import Input from 'material-ui/Input';
+import { withStyles } from 'material-ui/styles';
+
+const SearchPanelInputBase = ({
+  changeSearchValue, searchValue, getMessage, classes, ...restProps, className,
+}) => (<Input
+  className={classNames(classes.input, className)}
+  onChange={e => changeSearchValue({ searchValue: e.target.value })}
+  value={searchValue || ''}
+  placeholder={getMessage('searchPlaceholder')}
+  {...restProps}
+/>);
+
+SearchPanelInputBase.propTypes = {
+  changeSearchValue: PropTypes.func.isRequired,
+  searchValue: PropTypes.string,
```

Comment:
Not sure if we should use the 'search...' prefix in the property name. I believe we `value` would be quite clear in the context of the `SearchPanelInput` component.

Reply:
"searchValue" is more clear. I'd leave as is.